routes/car.js

```diff
@@ -1,8 +1,90 @@
 const express = require('express');
 const router = express.Router();
+const Car = require('../models/car');
+const User = require('../models/user');
+const jwtDecode = require('jwt-decode');
 
 router.get('/', (req, res) => {
-  res.send('car Route is working')
+    res.send('car Route is working')
 });
 
+
+router.post('/add', async (req, res) => {
+    const {_id} = jwtDecode(req.header('authorization'))['user'];
+
+    await User.findOne({_id: _id}).then(async (user) => {
+        if(user !== null || user !== 'undefined'){
+            await Car.findOne({createdBy: _id, license: req.body.license}).then(async (car) => {
+                if (car === null){
+                    const newCar = new Car(Object.assign({createdBy: _id}, req.body))
```

Comment:
i don't think need Object.assign
You can just do 
```suggestion
                    const newCar = new Car({ 
                         createdBy: _id,
                         ...req.body
                    })
```

Personally would prefer extracting fields out of req.body before passing in make it more readable